src/Tests/Creation/When_creating_queues.cs

```diff
@@ -453,5 +456,49 @@ await namespaceManager.CreateQueue(new QueueDescription("existingqueue")
             //cleanup
             await namespaceManager.DeleteQueue("existingqueue");
         }
+
+        [Test]
+        public async Task Should_not_update_properties_of_an_existing_shared_queue()
+        {
+            var queuePath = "errorQ";
+
+            var namespaceManager = new NamespaceManagerAdapter(NamespaceManager.CreateFromConnectionString(AzureServiceBusConnectionString.Value));
+            if (!await namespaceManager.QueueExists(queuePath).ConfigureAwait(false))
+            {
+                await namespaceManager.CreateQueue(new QueueDescription(queuePath)).ConfigureAwait(false);
+            }
+
+            var settings = new DefaultConfigurationValues().Apply(new SettingsHolder());
+            settings.Set("errorQueue", queuePath);
+            settings.Set("TypesToScan", new List<Type> {typeof(ConfigureAuditQueue)});
```

Comment:
Problem is that `GetConfigSection` assumes that there are "available types" in the settings. We could make that a TryGetAvailableTypes or just make it return new List<Type>

https://github.com/Particular/NServiceBus/blob/release-6.0.0/src/NServiceBus.Core/SettingsExtensions.cs#L20

@Particular/nservicebus-maintainers thoughts?